### DISCUSSION TITLE
Storing BLOBs

### DISCUSSION THREAD

**Discussion by miQSH** [sqlitebrowser/sqlitebrowser#3294] [General]
Hello,

I am new to DB Browser for SQLite and would like to store a BLOB into a fresh database. I have the fields "ID" (primary key), "name", and "pickob". In "pickob", I would like to store a binary data object, from the DB Browser. I can't find any option to load any binary object into that field, however.

How is this done?

Thank you.

Regards

**Comment by scottfurry** [comment]
Do you have a hex viewer application? This would be some other external application. Open file in hex viewer, copy contents, and then paste into the cell in DB4S. That seems the *easiest* path.

An operating system wasn't given so I have no suggestions for hex viewer application.

**Reply by miQSH** [reply]
Thanks. I think I will be able to install one. I am using Ubuntu.

**Reply by miQSH** [reply]
On closer inspection, I found that DB Browser has a button for loading binary data.
